test/unit/stateView/document/query/findConflictingConditions.spec.js

```diff
@@ -0,0 +1,49 @@
+const findConflictingConditions = require('../../../../../lib/stateView/document/query/findConflictingConditions');
+
+const allowedPairsTestCases = [
+  ['<', '>'],
+  ['<', '>='],
+  ['>', '<'],
+  ['>', '<='],
+];
+const restrictedPairsTestCases = [
+  ['==', '<'],
+  ['<', '<'],
+  ['>', '>'],
+  ['=>', '<='],
+  ['==', '=='],
```

Comment:
Don't we need at least one test with non-range comparison operators like "contains" for example?